tests/postgres_tests/test_bulk_create.py

```diff
@@ -0,0 +1,33 @@
+from datetime import date
```

Comment:
`postgres_tests` is for testing `django.contrib.postgres`. Use `tests/bulk_create` and `@skipUnlessDBFeature('can_return_rows_from_bulk_insert')`.

Reply:
Oh, I didn't know that. I'll use `tests/bulk_create` instead.